QualityReport.wdl

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Sequencing Analysis Support Core - Leiden University Medical Center
+
+import "tasks/fastqc.wdl" as fastqc
+import "tasks/biopet.wdl" as biopet
+
+workflow QualityReport {
+    File read
+    String outputDir
+    String? extractAdaptersOutput = outputDir + "/extractAdapters"
+    String? fastqcOutput = outputDir + "/fastqc"
+    Boolean? extractAdapters = false
```

Comment:
why this is default false?

Reply:
Because this invokes some extra tasks. The extraction of adapters is not needed for the QC only for possible later steps in the pipeline. Since it is not necessary for the quality reports it is disabled by default.
Alternative: make this a non-optional so it always has to be specified.